Packs/Jira/Integrations/JiraV3/JiraV3.yml

```
@@ -1460,6 +1460,56 @@ script:
       name: comment
     description: Use this command to create a link between two issues.
     name: jira-issue-to-issue-link
+  - arguments:
+    - description: The issue ID.
+      name: issue_id
+      required: true
+    description: Fetches questions and answers for forms tied to a Jira issue.
+    name: jira-issue-get-forms
+    outputs:
+    - contextPath: JiraForms.ID
+      description: The ID of the form.
+      type: Unknown
+    - contextPath: JiraForms.Issue
+      description: Issue the form belongs to.
+      type: Unknown
+    - contextPath: JiraForms.Questions
+      description: Questions for the form.
+      type: Unknown
+  - arguments:
+    - description: The user's key (also called ID).
+      name: key
+      required: false
+    - description: The user's username.
+      name: username
+      required: false
+    description: Get's information about the specified user.
+    name: jira-get-user
```

I'd give it a more informative name

```suggestion
    name: jira-get-user-info
```

---

There is already a `get_user_info` function in the class which only gets information for the user account doing the API calls. This isn't an integration command. I can merge them together and set an argument for `api_user=False` and in the few calls to the current function set that to True and grab the api user's information so it continues like normal.